package.json

```diff
@@ -33,6 +33,7 @@
         "pwa:start": "lerna run start --scope @padloc/pwa",
         "server:start": "lerna run start --scope @padloc/server --stream",
         "server:start-dry": "lerna run start-dry --stream --scope @padloc/server",
+        "electron:start": "lerna run start --scope @padloc/electron",
```

Comment:
FYI, I changed this script's name to `dev` in my current PR.

Reply:
I don't see it? It's still `start` after your merged PR, which makes sense since it's not really "watching", just building it once and showing locally.